pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -139,6 +139,22 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 		return nil, errors.WithStack(err)
 	}
 
+	// Retrieve storage class mapping information and update pvc StorageClassName with new name
+	storageClassMapping := make(map[string]string)
+	p.Log.Info("Retrieving storage class mapping information from configMap")
+	err = pluginItem.RetrieveStorageClassMapping(restConfig, veleroNs, storageClassMapping, p.Log)
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to retrieve storage class mapping information : %v", err)
```

Comment:
Are you returning error if mapping is not found?  Mapping is optional and most of time it is not needed.

Reply:
Change RetrieveStorageClassMapping so no error will be returned if no config map is found.